### DISCUSSION TITLE
能给个1.9.1版本的helm部署脚本吗？

### DISCUSSION THREAD

**Discussion by cyfanhu** [hhyo/Archery#1953] [Q&A]
能给个1.9.1版本的helm部署脚本吗？

**Comment by tangguangliang** [comment]
大佬，你用helm部署成功了吗，我也想在k8s环境中部署，启动成功之后配置https证书一直报错

**Reply by tangguangliang** [reply]
Forbidden (Origin checking failed - https://archery does not match any trusted origins.): /authenticate/

**Reply by cyfanhu** [reply]
部署成功了哦，没遇到过证书问题。

**Comment by LeoQuote** [comment]
https://github.com/hhyo/Archery/wiki/k8s @tangguangliang 参考下这个 wiki, 刚写的
